public/manifest.json

```diff
@@ -1,15 +1,105 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "IPFS-Share",
+  "name": "IPFS Share",
   "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
+      "src": "/android-chrome-192x192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "/android-chrome-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
     }
   ],
-  "start_url": "./index.html",
+  "theme_color": "#ffffff",
+  "background_color": "#ffffff",
+  "start_url": "/",
   "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "share_target": {
+    "action": "/upload",
+    "method": "POST",
```

Comment:
Hm.. are we planning to handle POST via Service Worker?
If looking into this takes too much time we can extract it into a separate issue. 

Having PWA mode, even without share target is already a win.

Reply:
I'm trying to test this out, but is hard since JS-IPFS requires https and I have to test with my phone, but will update https://github.com/ipfs-shipyard/ipfs-share-files/issues/106 with news 👍

Reply:
Removed the share for now, please approve the rest 🙏